### PR TITLE
Upgrade pnpm to 9

### PR DESCRIPTION
**Pull request by manuelblum**
Also, use PNPM from Corepack.

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -7,7 +7,8 @@ cd $(dirname $0)
 # use correct node version and install dependencies
 nvm install
 nvm use
-npm i -g pnpm@8
+corepack enable pnpm
```
Comment (kaufmo) [inline install.sh:10]:
```suggestion
corepack enable
```
this is enough because you set the package manager in package.json file :+1:

Comment (manuelblum) [reply install.sh]: done

File: package.json [hunk 2]
```diff
@@ -48,10 +48,10 @@
         "typescript": "^4.2.2",
         "wait-on": "^5.3.0"
     },
-    "packageManager": "pnpm@8.15.5",
+    "packageManager": "pnpm@9.13.2+sha512.88c9c3864450350e65a33587ab801acf946d7c814ed1134da4a924f6df5a2120fd36b46aab68f7cd1d413149112d53c7db3a4136624cfd00ff1846a0c6cef48a",
```
Comment (kaufmo) [inline package.json:51]: the sha here is also a bit overkill I think version alone should be enough

Comment (manuelblum) [reply package.json:51]: was automatically added, yeah ofc we should be able to delete it

**Review by kaufmo — APPROVED** [review]

**Review by johnnyomair — CHANGES_REQUESTED** [review]
Please fix conflicts

**Review by johnnyomair — COMMENTED** [review]
We need to decide if we want to use Corepack first.

**Comment by manuelblum** [comment]
@johnnyomair, we decided to go with this solution.

**Review by kaufmo — APPROVED** [review]

**Review by johnnyomair — APPROVED** [review]

**Pull request merged**